factory/builder.py

```diff
@@ -335,14 +332,18 @@ def __getattr__(self, name):
         """
         if name in self.__pending:
             raise errors.CyclicDefinitionError(
-                "Cyclic lazy attribute definition for %r; cycle found in %r." %
-                (name, self.__pending))
+                "Cyclic lazy attribute definition for %r; cycle found in %r."
+                % (name, self.__pending)
+            )
         elif name in self.__values:
             return self.__values[name]
         elif name in self.__declarations:
             declaration = self.__declarations[name]
             value = declaration.declaration
-            if enums.get_builder_phase(value) == enums.BuilderPhase.ATTRIBUTE_RESOLUTION:
+            if (
+                enums.get_builder_phase(value)
+                == enums.BuilderPhase.ATTRIBUTE_RESOLUTION
+            ):
```

Comment:
Because it exceeds the line size limit. An easy way to workaround these changes is by introducing an intermediary variable. Pushed commit 9e60d270a159d520b44c472331e0b61c6e0954f3 for an example.

Reply:
Another option is to increase the line-length for Black. That would avoid this change and help keeping logging lines on a single line.

Reply:
Increases the line length in the latest commit, removing the need for this line to be reformatted.